apps/src/sites/studio/pages/components/LevelTokenDetails.jsx

```diff
@@ -0,0 +1,153 @@
+/* global levelKeyList */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import VirtualizedSelect from 'react-virtualized-select';
+import 'react-virtualized/styles.css';
+import 'react-select/dist/react-select.css';
+import 'react-virtualized-select/styles.css';
+import _ from 'lodash';
+
+const levelKeyOptions = _.map(levelKeyList, (label, value) => ({label, value: +value}));
```

Comment:
Feels strange to have this so far away from the one place it's used. I imagine that the argument for not doing this directly in the render is that we avoid repeated work, which might make sense.